### PR TITLE
add redeeming docs

### PR DESCRIPTION
**Pull request by fernandezdavid**

### CODE REVIEW

File: pages/mainnet/validator-setup/redeeming-funds-and-retiring.mdx [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+Your node will cease to participate in auctions. You will have all remaining $FLIP credited to your balance upon the conclusion of the final auction. You will still need to follow the next steps to claim them.
+
+## Redemption Process in Auctions App 
+
+The redemption process is initiated via the [Chainflip Auctions App](https://auctions.chainflip.io/auctions) > **My Nodes** alongside the Chainflip CLI. You will need the address of the Ethereum account to which you want your $FLIP to be sent.
+
+In the Auctions App, select "Redeem". This will guide you through the process.
+
+You have two options. By default, the command will redeem the maximum available funds from your account, taking into account the bond. For example, imagine your account balance is 1000 \$FLIP and the bond is 800 \$FLIP, the following would withdraw 795 \$FLIP net of tax, to the Ethereum account `0xcf..cf`:
```
Comment (niklasnatter) [inline pages/mainnet/validator-setup/redeeming-funds-and-retiring.mdx:33]: i dont think 795 is right here. i think it should be 195?

File: pages/mainnet/validator-setup/redeeming-funds-and-retiring.mdx [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+Once you have completed the request, continue in the Auctions App, where the redemption should display.
+
+### Redemption Tax
+
+The network requires a **Redemption Tax of 5 $FLIP** per redemption. This is to reduce *redemption request spam*. These requests are expensive for the network to process, and the cost for this is now carried by the requester, via the tax.
+
+### Waiting Period
+
+A waiting period of **48 hours** before it can be used to redeem. This delay is a security measure to limit the risk in Chainflip's Vaults.
```
Comment (niklasnatter) [inline pages/mainnet/validator-setup/redeeming-funds-and-retiring.mdx:55]:
```suggestion
A waiting period of **48 hours** applies before a certificate can be used to redeem. This delay is a security measure to limit the risk in Chainflip's Vaults.
```

File: pages/mainnet/validator-setup/redeeming-funds-and-retiring.mdx [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+You have two options. By default, the command will redeem the maximum available funds from your account, taking into account the bond. 
+
+**Example:** 
+
+Imagine your account balance is **1000 \$FLIP** and the bond is **800 \$FLIP**. 
+
+You can redeem **200 \$FLIP** minus the tax, to the Ethereum account `0xcf..cf`: 
+
+**195 \$FLIP** net.
```
Comment (niklasnatter) [inline pages/mainnet/validator-setup/redeeming-funds-and-retiring.mdx:41]: lots of new line here now. not sure if that was intended

```suggestion

Imagine your account balance is **1000 \$FLIP** and the bond is **800 \$FLIP**.  You can redeem **200 \$FLIP** minus the tax, to the Ethereum account `0xcf..cf`: **195 \$FLIP** net.
```

Comment (fernandezdavid) [reply pages/mainnet/validator-setup/redeeming-funds-and-retiring.mdx:41]: that format is intented :D

**Review by acdibble — APPROVED** [review]

**Pull request merged**